Rocket.Chat/Views/Cells/Chat/ChatMessageCell.swift

```diff
@@ -346,6 +364,8 @@ extension ChatMessageCell: UIGestureRecognizerDelegate {
 extension ChatMessageCell {
 
     override func awakeFromNib() {
+        super.awakeFromNib()
```

Comment:
According to docs we should pretty much always call it since many UIKit classes provides some additional actions there.

Reply:
Great finding!